distribution/ddf-common/src/main/resources-filtered/etc/application-definitions/search-ui.json

```diff
@@ -1,10 +1,8 @@
 {
   "name": "search-ui-app",
-  "description": "The Search UI is an application that not only provides results in a html format but also provides a convenient, simple querying user interface.\nThe left pane of the SearchUI contains basic fields to query the Catalog and other Sources. The right pane consists of a map.\nIt includes both standard (3d globe) and simple (text page) versions.::Search UI",
+  "description": "The Search UI is an application that not only provides results in a html format but also provides a convenient, simple querying user interface.::Search UI",
   "bundleLocations": [
-    "mvn:ddf.ui.search/search-endpoint/${project.version}",
     "mvn:ddf.ui.search/search-redirect/${project.version}",
-    "mvn:ddf.ui.search/standard/${project.version}",
-    "mvn:ddf.ui/catalog-ui-search/${project.version}"
+    "mvn:ddf.ui.search/simple/${project.version}"
```

Comment:
There aren't any configs for this bundle right now, but any future configs should belong it this app. It doesn't hurt to have this here now, and I think it makes the purpose of this file clearer and more maintainable.